src/tutorials/0005/06-exercise.md

```diff
@@ -0,0 +1,3 @@
+Use `ls` to list the contents of the root directory you created in the previous exercise.
+
+**Hint:** The CID of the root directory ( `""` ) has been stored for you in the `rootCID` variable in the starter code below. You could have jotted this value down after the last exercise, but here we're looking through the array returned by the `add` method to find the element with the empty string as its `path` (the root directory) and saving its `hash` value, or CID.
```

Comment:
> You could have jotted this value down after the last exercise, but here we're looking at the last element (the wrapping directory) in the array returned by the `add` method and saving its `hash` value, or CID.

Do you think this would be better?

Reply:
I'm fine with changing the way we search for it, but I don't think that's quite enough context to show that the last entry is ALWAYS the wrapping directory which is key to presenting this as a reasonable way to approach it. 

But regardless, I don't think this addresses the problem we were trying to anticipate. If you added some files using `{ wrapWithDirectory: true }` and later wanted to find them, how would you know where they were? Our statement was meant to be more about "write this down right now because you'll never see it except in the results of `add`." Is that incorrect? 

If I do `add`, wrapping with directories, 5 times, it creates 5 different I'm-not-calling-them-root directories. If I need to go back and retrieve a specific one of them later, while I don't have access to the results of my original `add` method, how do I refer to it?